libraries/AP_Airspeed/AP_Airspeed.cpp

```diff
@@ -887,6 +886,36 @@ bool AP_Airspeed::get_hygrometer(uint8_t i, uint32_t &last_sample_ms, float &tem
 }
 #endif // AP_AIRSPEED_HYGROMETER_ENABLE
 
+#if HAL_GCS_ENABLED
+void AP_Airspeed::send_mavlink_airspeed(mavlink_channel_t chan)
+{
+    for (uint8_t i=0; i<AIRSPEED_MAX_SENSORS; i++) {
```

Comment:
Never loop to send bulk messages out like this for a sensor, we need to stash the last sent id, and send exactly one additional one. You can see an example of handling this here: https://github.com/ArduPilot/ardupilot/pull/14592

The problem is on a highly used link you will never see the second airspeed instance, and will only receive the first driver.

Reply:
Now rotates through sensors and only sends one per call.